### PR TITLE
Housekeeping: amend user invitation case for members controller

### PR DESCRIPTION
**Pull request by machisuji**
Adds missing spec for #14717.

### CODE REVIEW

File: spec/controllers/members_controller_spec.rb [hunk 1]
```diff
@@ -175,6 +175,52 @@
...
+          expect(users).to all be_member_of(project)
+
+          expect(Member.count).to eq (original_member_count + users.size)
+        end
+
+        it 'invites new users' do
+          expect(invited_users.size).to be 2
+
+          invited_users.each do |user|
+            expect(Mails::InvitationJob)
```
Comment (machisuji) [inline spec/controllers/members_controller_spec.rb]: I've tried to check via `ActionMailer::Base.deliveries` and also to check for enqueued delayed jobs, but both is empty here for some reason. If anyone knows why, please let me know. I don't. So I just test it like this instead.

Comment (aaron-contreras) [reply spec/controllers/members_controller_spec.rb]: You want to call `perform_enqueued_jobs` before the expectation.

File: spec/controllers/members_controller_spec.rb [hunk 2]
```diff
@@ -175,6 +175,52 @@
...
+        it 'adds members' do
+          expect(users.size).to eq 4 # 2 emails, 2 existing users
+          expect(users).to all be_member_of(project)
+
+          expect(Member.count).to eq (original_member_count + users.size)
+        end
+
+        it 'invites new users' do
+          expect(invited_users.size).to be 2
+
```
Comment (aaron-contreras) [inline spec/controllers/members_controller_spec.rb]:
```suggestion
          expect(invited_users.size).to be 2

          perform_enqueued_jobs

```

**Review by aaron-contreras — APPROVED** [review]
Thanks for this 👍  It's fine either way but if you'd like to test on the actual mail delivery I annotated how you can do so.

File: spec/controllers/members_controller_spec.rb (hunk 2 above)
Comment (machisuji) [reply spec/controllers/members_controller_spec.rb]: @aaron-contreras Thanks, that would help. 😄

**Comment by machisuji** [comment]
> Thanks for this 👍 It's fine either way but if you'd like to test on the actual mail delivery I annotated how you can do so.

Thanks a lot! I keep forgetting this and didn't seem to see it in the other specs I looked up either. Anyway, I'll adapt the PR to simply use that and then check the deliveries.

File: spec/controllers/members_controller_spec.rb (hunk 1 above)
Comment (machisuji) [reply spec/controllers/members_controller_spec.rb]: Note to self: `perform_enqueued_jobs`

**Pull request merged**